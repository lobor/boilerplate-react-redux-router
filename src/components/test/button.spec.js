import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Button from './../button'

test('render button', () => {
  const tree = renderer.create(<Button />).toJSON()
  expect(tree).toMatchSnapshot()
})
