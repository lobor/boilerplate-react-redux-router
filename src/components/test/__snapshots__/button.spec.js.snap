// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render button 1`] = `
.c0 {
  display: inline-block;
  height: 38px;
  padding: 0 30px;
  color: #555;
  text-align: center;
  font-size: 11px;
  font-weight: 600;
  line-height: 38px;
  -webkit-letter-spacing: .1rem;
  -moz-letter-spacing: .1rem;
  -ms-letter-spacing: .1rem;
  letter-spacing: .1rem;
  text-transform: uppercase;
  text-decoration: none;
  white-space: nowrap;
  background-color: transparent;
  border-radius: 4px;
  border: 1px solid #D1D1D1;
  cursor: pointer;
  box-sizing: border-box;
  margin-left: 10px;
}

.c0:hover {
  color: #333;
  border-color: #888888;
  outline: 0;
  background-color: inherit;
}

.c0:first-child {
  margin-left: 0;
}

@media (max-width:25em) {
  .c0 {
    margin-left: 0;
    margin-bottom: 10px;
  }

  .c0:last-child {
    margin-bottom: 0;
  }
}

<button
  className="c0"
/>
`;
